hail/python/hail/docs/conf.py

```diff
@@ -19,8 +19,17 @@
 
 import datetime
 import os
-import re
 import sys
+from importlib.util import module_from_spec, spec_from_file_location
+
+
+def __load_module(name, path):
+    spec = spec_from_file_location(name, path)
+    mod = module_from_spec(spec)
+    sys.modules[name] = mod
+    spec.loader.exec_module(mod)
+    return mod
```

Comment:
This is arguably the only more complicated bit about this change. I could have used a regex to read extract this information but settled on this as I believe it would fail if we changed versioning in the future and forgot to modify this.